emojivoto-web-app/js/setup_dev_env.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash
+
+# global vars
+CONTAINER_ID=''
+OS=''
+ARCH=''
+MOUNT_VOLUME_LOCAL=''
+USE_TELEMETRY=
+ACTIVITY_REPORT_TYPE='INTERMEDIATE_CLOUD_TOUR'
```

Comment:
we need to add this activity report type in saas_app.

Reply:
🤔 not sure if we should use a better name to make clear that these metrics are coming from the script. Maybe something like `INTERMEDIATE_CLOUD_TOUR_SCRIPT`, thinking in the other tours, we usually have:
_START
_TRACKING
_END
Maybe for this would make sense to have also the _SCRIPT
It is just an idea

Reply:
I created the PR to add the activity type in saas_app, https://github.com/datawire/saas_app/pull/2487 I used `INTERMEDIATE_CLOUD_TOUR_SCRIPT` but is a draft PR so I'm open to changing it.